### PR TITLE
Massage the bazelrc

### PR DESCRIPTION
**Pull request by lalten**
Split off https://github.com/mvukov/rules_ros2/pull/238

### CODE REVIEW

File: .github/workflows/ci.bazelrc [hunk 1]
```diff
@@ -22,9 +22,7 @@ build:remote --noslim_profile
 
 # Don't spam CI logs.
 build --show_progress_rate_limit=60
+common --color=yes
 
-# Don't rely on test logs being easily accessible from the test runner,
-# though it makes the log noisier.
-test --test_output=errors
```
Comment (mvukov) [inline .github/workflows/ci.bazelrc:28]: I think this is fine for CI.

File: .bazelrc [hunk 2]
```diff
@@ -37,6 +41,11 @@ build --noexperimental_check_output_files
 # Don't bother building targets which aren't dependencies of the tests.
 test --build_tests_only
 
+# Show all the problems
+test --keep_going
+test --test_output=errors
+build --verbose_failures
```
Comment (mvukov) [inline .bazelrc:47]: I'm fine with `--test_output=errors` being here but the other two I don't see necessary by default (e.g. I use them on occasions, only).

File: .bazelrc [hunk 3]
```diff
@@ -16,6 +16,10 @@ build --noenable_bzlmod
 # This is a mandatory flag.
 build --incompatible_default_to_explicit_init_py
 
+# Don't let local Python site packages leak into the build and cause problems
+common --incompatible_strict_action_env
```
Comment (mvukov) [inline .bazelrc:20]: This one is already in this file, perhaps change that one from build to common.

File: .bazelrc [hunk 4]
```diff
@@ -16,6 +16,10 @@ build --noenable_bzlmod
...
+common --action_env=PYTHONNOUSERSITE=1
```
Comment (mvukov) [inline .bazelrc:21]: Very nice, thanks! :)

File: .bazelrc (hunk 2 above)
Comment (lalten) [reply .bazelrc]: removed

File: .github/workflows/ci.bazelrc (hunk 1 above)
Comment (lalten) [reply .github/workflows/ci.bazelrc:28]: removed

File: .bazelrc (hunk 3 above)
Comment (lalten) [reply .bazelrc]: changed

**Review by mvukov — APPROVED** [review]

**Pull request merged**